erpnext/crm/doctype/email_campaign/email_campaign.py

```
@@ -77,58 +80,129 @@ def update_status(self):
+				doctype="Email Campaign",
+				name=campaign_name,
+				subject=subject,
+				content=content,
+				sender=sender,
+				bcc=batch,
+				communication_medium="Email",
+				sent_or_received="Sent",
+				send_email=True,
+				email_template=email_template.name,
+			)
+		except Exception:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)